CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 ## [Unreleased]
 ### Fixed
 - item quantity in shipping mapping
+- saving of the address in customer registration
```

Comment:
capitalized, may as well change the other one too :)

Reply:
done